PKGBUILDS/phosh/phoc/PKGBUILD

```
@@ -1,17 +1,16 @@
 # Maintainer: Danct12 <danct12@disroot.org>
 pkgname=phoc
-pkgver=0.26.0
+pkgver=0.27.0
 pkgrel=1
 pkgdesc="wlroots based Wayland compositor for mobile phones"
 arch=('x86_64' 'armv7h' 'aarch64')
 url="https://gitlab.gnome.org/World/Phosh/phoc"
 license=('GPL')
-depends=('gobject-introspection' 'gnome-desktop' 'libinput' 'seatd' 'mutter'
-         'xcb-util-errors' 'xcb-util-renderutil' 'xcb-util-wm' 'wayland-protocols'
-         'libvulkan.so')
-makedepends=('git' 'meson' 'ninja' 'glslang' 'vulkan-headers')
+depends=('gobject-introspection' 'gnome-desktop' 'libinput' 'seatd' 'mutter' 'libxcb'
+         'xcb-util-errors' 'xcb-util-renderutil' 'xcb-util-wm' 'wayland-protocols' 'libvulkan.so')
+makedepends=('git' 'meson' 'ninja' 'glslang' 'vulkan-headers' 'wayland-protocols')
```

```suggestion
makedepends=('git' 'meson' 'ninja' 'glslang' 'vulkan-headers')
```
https://wiki.archlinux.org/title/PKGBUILD#makedepends